### ISSUE TITLE
Docker Support - MIVisionX in Ubuntu20.04

### ISSUE THREAD

**Issue by hyongtao** [GPUOpen-ProfessionalCompute-Libraries/MIVisionX#552]
I have followed the instructions of MIVisionX docker  
as the page: https://gpuopen-professionalcompute-libraries.github.io/MIVisionX/#docker

```
$ docker images | grep mivisionx
mivisionx/ubuntu-18.04                                               latest                            d60640ccb01b   11 months ago   14.6GB
mivisionx/ubuntu-16.04                                               tutorial                          d7c1d5f26185   2 years ago     29.3GB

```
But when I run the docker, I got thses error:
```
$ sudo docker run -it --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-18.04
docker: Error response from daemon: OCI runtime create failed: invalid mount {Destination::/root/.Xauthority Type:bind Source:/var/lib/docker/volumes/8d2d2d27926f9057e07c933e95da8c61497240fdb984b3e1ba42b7def5a034c1/_data Options:[rbind]}: mount destination :/root/.Xauthority not absolute: unknown.
ERRO[0002] error waiting for container: context canceled
```
Could you help me out? Thanks a lot.

**Comment by kiritigowda** [comment]
@hyongtao thanks for reporting this issue. This seems to be an issue with the image being outdated. I will update those images now. In the mean time you can also build the updated images using the dockerfiles in docker folder.

```
sudo docker build -f MIVisionX/docker/mivisionx-on-ubuntu18.dockerfile -t mivisionx-18.04:latest .
```

**Comment by kiritigowda** [comment]
@hyongtao - The docker file is updated and this should work now

```
sudo docker run -it --device=/dev/kfd --device=/dev/dri --cap-add=SYS_RAWIO --device=/dev/mem --group-add video --network host mivisionx/ubuntu-18.04
root@ThreadRipperUbuntu:/# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.5 LTS"
```
Let me know if this fixes your issue and we can close this issue.

**Comment by kiritigowda** [comment]
@hyongtao any updates?

**Comment by hyongtao** [comment]
Hi Kirit Gowda,
Thanks for your help. I will test the dockerfile this week. Nothing update.

From
Yongtao Huang
________________________________
发件人: Kiriti Gowda ***@***.***>
发送时间: 2021年7月13日 8:23
收件人: GPUOpen-ProfessionalCompute-Libraries/MIVisionX ***@***.***>
抄送: Yongtao Huang ***@***.***>; Mention ***@***.***>
主题: Re: [GPUOpen-ProfessionalCompute-Libraries/MIVisionX] Docker Support - MIVisionX in Ubuntu20.04 (#552)


@hyongtao<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fhyongtao&data=04%7C01%7Chyongtao%40vmware.com%7Ce24b1a3587ab44d68ada08d945947724%7Cb39138ca3cee4b4aa4d6cd83d9dd62f0%7C0%7C0%7C637617326221625410%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=6xatv7Vutk7hGroWSLXjdemuvgEmLeqfrxaZSMkP7i8%3D&reserved=0> any updates?

―

**Comment by kiritigowda** [comment]
@hyongtao if this issue persists please reopen this issue. Fix #557
